src/game-master/containers/RoundPhaseButtonContainer.js

```diff
@@ -0,0 +1,23 @@
+import { h } from 'preact';
+import { connect } from 'preact-redux';
+import { createStructuredSelector } from 'reselect';
+
+import { Button } from '../../components/Button/Button';
+import { roundStarted } from '../../shared/selectors/round-phase-selectors';
+import { triggerRoundStart, triggerRoundEnd } from '../actions/gm-round-phase-actions';
+
+const RoundPhaseButtonContainer = connect(createStructuredSelector({
+    roundStarted,
+}), {
+    triggerRoundStart,
+    triggerRoundEnd,
+})(({ roundStarted, triggerRoundStart, triggerRoundEnd }) => (
+    <Button
+        iconLeft={roundStarted ? 'stop' : null }
+        iconRight={!roundStarted ? 'arrow-right' : null }
+        text={roundStarted ? 'STOP LEVEL' : 'NEXT LEVEL'}
+        onClick={roundStarted ? () => triggerRoundEnd() : () => triggerRoundStart() }
```

Comment:
Is wrapping of AC in additional function necessary?

Reply:
without this wrapper payload of created action is `MouseEvent` 👎

Reply:
:(